crates/metaslang/bindings/src/lib.rs

```diff
@@ -471,12 +487,20 @@ impl<'a, KT: KindTypes + 'static> Reference<'a, KT> {
             .expect("Reference does not have a valid file descriptor")
     }
 
-    pub fn jump_to_definition(&self) -> Result<Definition<'a, KT>, ResolutionError<'a, KT>> {
-        Resolver::build_for(self).first()
+    pub fn resolve_definition(&self) -> Result<Definition<'a, KT>, ResolutionError<'a, KT>> {
```

Comment:
Q: would we ever use `resolve_definition()` here?
IIUC, we always provide all defs to the user, and let them decide how to rank them for now. Should it be removed?

Reply:
We discussed this offline and we'll remove this and related code (bindings compilation context, linearisation and results ranking), but I'm thinking I'll do it in a future PR.